android/src/main/java/com/lwansbrough/RCTCamera/RCTCameraViewManager.java

```diff
@@ -37,6 +37,11 @@ public void setType(RCTCameraView view, int type) {
         view.setCameraType(type);
     }
 
+    @ReactProp(name = "jpegQuality")
+    public void setJpegQuality(RCTCameraView view, int jpegQuality) {
+        view.setTorchMode(jpegQuality);
```

Comment:
I just copied original style. I'm not sure that I want to rewrite whole component.
About iOS implementation - I'm not working with iOS at this time.